<?xml version="1.0" encoding="UTF-8"?><testrun name="MainRunner" project="CucumberFramework" tests="12" started="12" failures="2" errors="1" ignored="0">
  <testsuite name="runner.MainRunner" time="8.993">
    <testsuite name="Feature: Login into account" time="8.993">
      <testsuite name="Scenario Outline: Login to account with credentials" time="8.993">
        <testsuite name="Examples:" time="8.993">
          <testsuite name="| http://www.webdriveruniversity.com/ | webdriver | webdriver | validation failed |" time="2.142">
            <testcase name="Given user navigates to &quot;http://www.webdriveruniversity.com/&quot;" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver | validation failed |" time="0.215"/>
            <testcase name="When user clicks on the login portal buton" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver | validation failed |" time="1.011"/>
            <testcase name="And user enters the &quot;webdriver&quot; username" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver | validation failed |" time="0.092"/>
            <testcase name="And user enter the &quot;webdriver&quot;" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver | validation failed |" time="0.077"/>
            <testcase name="When user clicks on the login button" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver | validation failed |" time="0.06"/>
            <testcase name="Then the user should be presented with the following prompt alert &quot;validation failed&quot;" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver | validation failed |" time="0.686"/>
          </testsuite>
          <testsuite name="| http://www.webdriveruniversity.com/ | webdriver | webdriver123 | validation successed |" time="1.234" incomplete="true">
            <error>java.lang.AssertionError: expected [validation successed] but found [validation succeeded]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at Steps.LoginSteps.the_user_should_be_presented_with_the_following_prompt_alert(LoginSteps.java:70)
	at ✽.Then the user should be presented with the following prompt alert "validation successed"(Login.feature:9)

org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : validation succeeded}
  (Session info: chrome=81.0.4044.138): validation succeeded
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T15:28:36.4Z'
System info: host: 'AnrinoiMac', ip: '192.168.1.147', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.15.4', java.version: '1.8.0_131'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{networkConnectionEnabled=false, chrome={chromedriverVersion=81.0.4044.138 (8c6c7ba89cc9453625af54f11fd83179e23450fa-refs/branch-heads/4044@{#999}), userDataDir=/var/folders/nt/nh0w0rrn36v9wlt_9bklsk8h0000gn/T/.com.google.Chrome.r8kzTT}, timeouts={implicit=0, pageLoad=300000, script=30000}, pageLoadStrategy=normal, unhandledPromptBehavior=dismiss and notify, strictFileInteractability=false, platform=MAC, proxy=Proxy(), goog:chromeOptions={debuggerAddress=localhost:51827}, acceptInsecureCerts=false, browserVersion=81.0.4044.138, browserName=chrome, javascriptEnabled=true, platformName=MAC, setWindowRect=true, webauthn:virtualAuthenticators=true}]
Session ID: 8e7484e0635cceb26bfd692a7d1457be
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:118)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:586)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:643)
	at org.openqa.selenium.remote.RemoteWebDriver$RemoteWebDriverOptions.deleteAllCookies(RemoteWebDriver.java:728)
	at Steps.LoginSteps.tearDown(LoginSteps.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at cucumber.runtime.Utils$1.call(Utils.java:40)
	at cucumber.runtime.Timeout.timeout(Timeout.java:16)
	at cucumber.runtime.Utils.invoke(Utils.java:34)
	at cucumber.runtime.java.JavaHookDefinition.execute(JavaHookDefinition.java:60)
	at cucumber.runtime.Runtime.runHookIfTagsMatch(Runtime.java:224)
	at cucumber.runtime.Runtime.runHooks(Runtime.java:212)
	at cucumber.runtime.Runtime.runAfterHooks(Runtime.java:206)
	at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:46)
	at cucumber.runtime.junit.ExecutionUnitRunner.run(ExecutionUnitRunner.java:102)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at cucumber.runtime.junit.ExamplesRunner.run(ExamplesRunner.java:59)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at cucumber.runtime.junit.ScenarioOutlineRunner.run(ScenarioOutlineRunner.java:53)
	at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:63)
	at cucumber.runtime.junit.FeatureRunner.runChild(FeatureRunner.java:18)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at cucumber.runtime.junit.FeatureRunner.run(FeatureRunner.java:70)
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:95)
	at cucumber.api.junit.Cucumber.runChild(Cucumber.java:38)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at cucumber.api.junit.Cucumber.run(Cucumber.java:100)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)

</error>
            <testcase name="Given user navigates to &quot;http://www.webdriveruniversity.com/&quot;" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver123 | validation successed |" time="0.505"/>
            <testcase name="When user clicks on the login portal buton" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver123 | validation successed |" time="0.547"/>
            <testcase name="And user enters the &quot;webdriver&quot; username" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver123 | validation successed |" time="0.07"/>
            <testcase name="And user enter the &quot;webdriver123&quot;" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver123 | validation successed |" time="0.074"/>
            <testcase name="When user clicks on the login button" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver123 | validation successed |" time="0.026"/>
            <testcase name="Then the user should be presented with the following prompt alert &quot;validation successed&quot;" classname="| http://www.webdriveruniversity.com/ | webdriver | webdriver123 | validation successed |" time="0.011">
              <failure>java.lang.AssertionError: expected [validation successed] but found [validation succeeded]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at Steps.LoginSteps.the_user_should_be_presented_with_the_following_prompt_alert(LoginSteps.java:70)
	at ✽.Then the user should be presented with the following prompt alert "validation successed"(Login.feature:9)

</failure>
            </testcase>
          </testsuite>
        </testsuite>
      </testsuite>
    </testsuite>
  </testsuite>
</testrun>
